src/Palette.elm

```diff
@@ -11,6 +13,71 @@ color =
     }
 
 
+{-| A box-shadow record, compatible both with `Element.layoutWith` Option and `Element.focusStyle`.
+
+For buttons, links and other interactables, we set focus styles
+We use a box-shadow instead of an outline, because it follows borders.
+
+Blue works well in this particular case.
+You might want to customise this for your application.
+
+Bear in mind that focus outlines should not be removed altogether,
+because they are used by people who rely on keyboard and/or screen reader navigation.
+They should also have a good color contrast with the background.
+
```

Comment:
Part of me really wants to throw in a link or two about focus styles here. I've had good results with people respecting them after they read a bit on them. What do you think?

Reply:
Yeah, absolutely! As a rule of thumb, I'm a big fan of self-describing code rather than comments to explain *what* code is doing, and, when needed, comments to explain *why* something unexpected is being done in code. This seems like a good start 👍